feature/google/src/main/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -22,5 +23,48 @@ fun SkuDetails.toStoreProduct() =
         introductoryPricePeriod.takeIf { it.isNotBlank() },
         introductoryPriceCycles,
         iconUrl,
-        JSONObject(originalJson)
+        JSONObject(originalJson),
+        getPricingPhases()
     )
+
+fun SkuDetails.getPricingPhases(): List<PricingPhase> {
+    if (subscriptionPeriod.isBlank()) {
+        return emptyList() // TODO what should we do for one time purchases?
```

Comment:
good question.. they don't change for BC5, so we could just pass null like we do for BC4...

Reply:
I think this'll depend on the design of the new API for StoreProduct - google clearly separates onetimepurchases from subscription purchases at an object level: https://developer.android.com/reference/com/android/billingclient/api/ProductDetails
I'd try to do the same in the new design and avoid mixing both. That'd mean one-time purchases don't even have pricing phases